helm_deploy/values-preprod.yaml

```diff
@@ -13,6 +13,10 @@ generic-service:
     HMPPS_AUTH_URL: https://sign-in-preprod.hmpps.service.justice.gov.uk/auth
     APP_MAGICLINK_URL: https://send-legal-mail-preprod.prison.service.justice.gov.uk/link/verify-link
 
+  allowlist:
+    bsi-1: "54.37.241.156/30"
+    bsi-2: "167.71.136.237/32"
+
```

Comment:
/30 allows 4 hosts, /32 allows 1

BSI did also request an IP6 range - 2001:41d0:800:715::/64
From what I can tell (though googling!), this seems to allow an incredible 18,446,744,073,709,551,616 hosts!!! So I've held back from allowing that until we can talk to then to get a better understanding of their requirement in this respect.